proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/MetatronResponse.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.ida.notification.contracts;
+
+import java.net.URI;
+
+public class MetatronResponse {
```

Comment:
I know this is a cool name but maybe something like `CountryMetadataInfo` would be more descriptive of what this DTO represents?

Reply:
Perhaps, but it is in keeping with the naming conventions we've used for other DTO's in the proxy node, like `EidasSamlParserResponse` and `HubResponseTranslatorRequest`

Reply:
I don't think those are particularly good names so would call this new DTO whatever is the best name, which I think is something specific and more descriptive